sklearn/svm/tests/test_svm.py

```diff
@@ -949,3 +949,53 @@ def test_decision_function_shape_two_class():
             clf = OneVsRestClassifier(estimator(
                 decision_function_shape="ovr")).fit(X, y)
             assert_equal(len(clf.predict(X)), len(y))
+
+
+def test_ovr_decision_function():
+    train_base_points = np.array([[1, 1]])
+
+    # For all the quadrants (classes)
+    X = np.vstack((
+        train_base_points * [1, 1],    # Q1
+        train_base_points * [-1, 1],   # Q2
+        train_base_points * [-1, -1],  # Q3
+        train_base_points * [1, -1]    # Q4
+        ))
+
+    Y = [0, 1, 2, 3]
```

Comment:
We generally have the convention of 

X for data
y for single target
Y multi target

So it would be nice to use capital `X` and small `y` in this test...